Dockerfile

```diff
@@ -14,11 +14,10 @@
 
 
 
-FROM golang:1.13.4-alpine3.10 as builder
+FROM golang:1.13.4 as builder
```

Comment:
any specific reason for not using 1.14?

Reply:
nope I just wasn't sure if we were ready to change it yet. Sometimes the new kubekins images don't have the newer golang versions built in

Reply:
Thanks for the clarification